engines/director/lingo/lingo-builtins.cpp

```diff
@@ -1220,7 +1220,7 @@ void LB::b_openXlib(int nargs) {
 		}
 	}
 
-	xlibName = d.asString();
+	xlibName = getFileName(pathMakeRelative(d.asString()));
```

Comment:
I think it should be just `getFileName(d.asString())` because we do not need to _search_ for this file.